README.md

```diff
@@ -54,7 +54,7 @@ section of this file tells the builder how to build and package your source. In
 
 | Option Name | Type | Default | Description |
 |----------|------|---------|-------------|
-| jdk | string | openjdk8 | Select the JDK used in the generated image. The only supported value is `openjdk8`.
+| jdk | string | openjdk8 | Select the JDK used in the generated image. Must be either `openjdk8` or `openjdk9`.
```

Comment:
Should we add a note somewhere here that `openjdk9` option cannot be combined with the server option?